assets/js/builder.index.entity.databasetable.js

```diff
@@ -257,6 +270,42 @@
         return result
     }
 
+    DatabaseTable.prototype.addIdColumn = function($target) {
+        var existingColumns = this.getColumnNames($target),
+            added = false
+
+        if ($.inArray('id', existingColumns) == -1) {
+            var tableObj = this.getTableControlObject($target),
+                currentData = this.getTableData($target),
+                rowData = {
+                    name: 'id',
+                    type: 'integer',
+                    unsigned: true,
+                    auto_increment: true,
+                    primary_key: true,
+                }
+
+            if (currentData.length-1) {
+                tableObj.addRecord('bottom', true)
+            }
+
+            tableObj.setRowValues(currentData.length-1, rowData)
```

Comment:
@DanHarrin  Spaces needed:
```suggestion
            tableObj.setRowValues(currentData.length - 1, rowData)
```